spec/requests/posts_spec.rb

```diff
@@ -1,20 +1,20 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    #    user = User.create('name' => 'Amy', 'bio' => 'bio', 'photo' => 'Tom.png', 'posts_counter' => 0)
-    before(:each) { get '/users/68/posts' }
+# RSpec.describe 'Posts', type: :request do
+#   describe 'GET /index' do
+#     #    user = User.create('name' => 'Amy', 'bio' => 'bio', 'photo' => 'Tom.png', 'posts_counter' => 0)
+#     before(:each) { get '/users/68/posts' }
 
-    it 'checks if action returns correct response status' do
-      expect(response).to have_http_status(:ok)
-    end
+#     it 'checks if action returns correct response status' do
+#       expect(response).to have_http_status(:ok)
+#     end
 
-    it 'checks if action rendered a correct template' do
-      expect(response).to render_template('index')
-    end
+#     it 'checks if action rendered a correct template' do
+#       expect(response).to render_template('index')
+#     end
 
-    it 'checks if correct placeholder is shown' do
```

Comment:
- [ ] I would recommend removing all the commented code in different files if you are no longer using them.

Reply:
Thank you for your feedback. I'll fix it.